docs/csharp/index.yml

```diff
@@ -1,162 +1,338 @@
-            url: linq/index.md
-          - text: "Versioning"
-            url: ./versioning.md
-            
-  - title: "What's new"
-    linkLists:
-      - linkListType: whats-new
-        links:
-          - text: "What's new in C# 13"
-            url: whats-new/csharp-13.md
-          - text: "What's new in C# 12"
-            url: whats-new/csharp-12.md
-          - text: "What's new in C# 11"
-            url: whats-new/csharp-11.md
-          - text: "What's new in C# 10"
-            url: whats-new/csharp-10.md
-      - linkListType: tutorial
-        links:
-          - text: Explore record types
-            url: whats-new/tutorials/records.md
-          - text: Explore top level statements
-            url: whats-new/tutorials/top-level-statements.md
-          - text: Explore new patterns
-            url: whats-new/tutorials/patterns-objects.md
-          - text: Write a custom string interpolation handler
-            url: whats-new/tutorials/interpolated-string-handler.md
-      - linkListType: reference
-        links:
-          - text: Breaking changes in the C# compiler
-            url: whats-new/breaking-changes.md
-          - text: Version compatibility
-            url: whats-new/version-update-considerations.md
-            
-  - title: "C# language reference"
-    linkLists:
-      - linkListType: reference
-        links:
-          - text: "Language reference"
-            url: language-reference/index.md
-          - text: "C# keywords"
-            url: language-reference/keywords/index.md
-          - text: "C# operators and expressions"
-            url: language-reference/operators/index.md
-          - text: Configure language version
-            url: language-reference/configure-language-version.md
-          - text: "C# language specification - C# 7 draft in progress"
-            url: ../../_csharpstandard/standard/README.md
 
-  - title: "Stay in touch"
-    linkLists:
-      - linkListType: reference
-        links:
-          - text: ".NET developer community"
-            url: https://dotnet.microsoft.com/platform/community
-          - text: "YouTube"
-            url: https://www.youtube.com/dotnet
-          - text: "Twitter"
-            url: https://twitter.com/DotNet
+# productDirectory section (Optional; Remove if not applicable.)
+productDirectory:
+  title: Learn to program with C#
+  summary: Major concepts and features of the C# language
+  items:
+    # Card
+    - title: Get started with C#
+      # imageSrc should be square in ratio with no white space
+      imageSrc: ../media/logo_csharp.png
+      links:
+        - url: tour-of-csharp/index.md
+          text: A tour of C#
+        - url: ./tour-of-csharp/tutorials/index.md
+          text: Beginner C# tutorials
+        - url: ./tour-of-csharp/tutorials/hello-world.yml
+          text: Try C# in your browser
+        - url: ./fundamentals/program-structure/index.md
+          text: "Inside a C# program"
+
```

Comment:
Can we add the learning resources here as well?
- C# for Beginners video series: https://aka.ms/dotnet/beginnervideos/learn/csharp
- Foundational C# Certification: https://aka.ms/csharp-certification